Remove extra newline at end of WPT file for WPT Exporter testing

jeffcarp opened the pull request:
Remove extra newline at end of WPT file for WPT Exporter testing

Bug: 700092
Change-Id: I60d34bfc70741f03bff10453ce0934631f0df0cd
Reviewed-on: https://chromium-review.googlesource.com/479676
Cr-Commit-Position: refs/heads/master@{#471374}
WPT-Export-Revision: dfd9f38d2f9735ea5f5e7b4de7ca9a95c6e025c6

<!-- Reviewable:start -->

<!-- Reviewable:end -->

w3c-bots commented:
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/231374402)

# Firefox (nightly) #
Testing web-platform-tests at revision 50a5261949e4320709c5c16475e125b804fbf254
Using browser at version BuildID 20170511133312; SourceStamp 838652a84b76c273e084d0705f3f4f3be89520a8
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>3 tests ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5898/workers/semantics/multiple-workers/001.html">/workers/semantics/multiple-workers/001.html</a></summary>


|   Subtest   | Results | Messages |
|-------------|---------|----------|
|             | OK      |          |
| `dedicated` | PASS    |          |
| `shared`    | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5898/workers/semantics/xhr/003.html">/workers/semantics/xhr/003.html</a></summary>


|                 Subtest                 | Results | Messages |
|-----------------------------------------|---------|----------|
|                                         | OK      |          |
| `async XMLHttpRequest in shared worker` | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5898/workers/semantics/xhr/004.html">/workers/semantics/xhr/004.html</a></summary>


|                Subtest                 | Results | Messages |
|----------------------------------------|---------|----------|
|                                        | OK      |          |
| `sync XMLHttpRequest in shared worker` | PASS    |          |

</details>
</details>

w3c-bots commented:
[View the complete job log.](https://travis-ci.org/w3c/web-platform-tests/jobs/231374409)

# Chrome (unstable) #
Testing web-platform-tests at revision 982f9142df71b91b3925c5172cd5dc6778848ecb
Using browser at version 60.0.3095.5 dev
Starting 10 test iterations
All results were stable
## All results ##
<details>
<summary>3 tests ran</summary>
<details>
<summary><a href="http://w3c-test.org/submissions/5898/workers/semantics/multiple-workers/001.html">/workers/semantics/multiple-workers/001.html</a></summary>


|   Subtest   | Results | Messages |
|-------------|---------|----------|
|             | OK      |          |
| `dedicated` | PASS    |          |
| `shared`    | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5898/workers/semantics/xhr/003.html">/workers/semantics/xhr/003.html</a></summary>


|                 Subtest                 | Results | Messages |
|-----------------------------------------|---------|----------|
|                                         | OK      |          |
| `async XMLHttpRequest in shared worker` | PASS    |          |

</details>
<details>
<summary><a href="http://w3c-test.org/submissions/5898/workers/semantics/xhr/004.html">/workers/semantics/xhr/004.html</a></summary>


|                Subtest                 | Results | Messages |
|----------------------------------------|---------|----------|
|                                        | OK      |          |
| `sync XMLHttpRequest in shared worker` | PASS    |          |

</details>
</details>

jeffcarp commented:
FYI: this is part of a test for creating PRs from Gerrit CLs. Please don't merge this PR manually.

jeffcarp committed:
Remove extra newline at end of WPT file for WPT Exporter testing

Bug: 700092
Change-Id: I60d34bfc70741f03bff10453ce0934631f0df0cd
Reviewed-on: https://chromium-review.googlesource.com/479676
Cr-Commit-Position: refs/heads/master@{#471374}
WPT-Export-Revision: dfd9f38d2f9735ea5f5e7b4de7ca9a95c6e025c6
